app/views/jobs/interest.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1> Jobs By Level of Interest </h1>
+<table>
+<th> Job </th>
+<th> Level of Interest </th>
+
+<% Job.all.order(:level_of_interest).reverse.each do |job| %>
```

Comment:
avoid querying your db in your views 👍 

also - this logic could be encapsulated in a method on `Job`!